source/apps/change-deployment-models.txt

```diff
@@ -17,19 +17,14 @@ or switch an App deployed in the Eastern USA from AWS to Azure. For a
 list of all available options, see :ref:`Deployment Models & Regions
 <deployment-model>`.
 
-To move an existing App, you begin a  **Deployment Migration** that
+To move an existing App, you begin a **Deployment Migration** that
 automatically moves your App's data and configuration to the new
 configuration. The migration process is designed to be as seamless as
-possible but requires up to 30 minutes of downtime. You can
-monitor the migration status but no requests will be processed until the
+possible but requires 5 to 30 minutes of downtime. You can
+monitor the migration status, but no requests will be processed until the
 migration is complete. For more information, see :ref:`Deployment
 Migration Process <deployment-migration-process>`.
 
-If you connect to your App from a Realm SDK, you must update your SDK to
-a version that supports changing deployment models. If you change
-deployment models before upgrading, the SDK will not be able to connect
-and requests will fail.
-
 .. todo sync with ENG post https://jira.mongodb.org/browse/RCORE-1346 to determine minimum versions. Until then, users have to talk to support.
```

Comment:
Since I see minimum SDK versions listed, did this get done? can we remove this TODO?

Reply:
oops didn't notice that todo! will remove - thanks cory!